core/src/test/scala/unit/kafka/admin/ReassignPartitionsCommandTest.scala

```diff
@@ -288,6 +288,28 @@ class ReassignPartitionsCommandTest extends ZooKeeperTestHarness with Logging {
     assertEquals(3, propsCapture.getValues.size) //3 brokers
   }
 
+  @Test
+  def testPropagateErrorsInExecute(): Unit = {
+    try {
```

Comment:
We should use `intercept` or `assertThrows` instead of this pattern. Same in the other test.